bigframes/ml/compose.py

```diff
@@ -46,6 +46,28 @@
 )
 
 
+class SQLScalarColumnTransformer(base.BaseTransformer):
+    def __init__(self, sql: str, target_column="transformed_{0}"):
+        super().__init__()
+        self.sql = sql
+        self.target_column = target_column
```

Comment:
Are these intended to be public properties? If so, could you make them private here and add a proper `@property` getter for each.

If not, please make these "private"

```suggestion
        self._sql = sql
        self._target_column = target_column
```